src/index.ts

```diff
@@ -1,9 +1,11 @@
 import express from 'express'
+import 'dotenv/config'
+import { PORT } from './config/dotenv'
 const app = express()
-const port = process.env.PORT || 3003
```

Comment:
I think this alternative port (3003, in this case) is important and should remain.